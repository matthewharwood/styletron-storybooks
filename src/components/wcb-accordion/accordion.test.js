// Link.react.test.js
import React from 'react';
import {Accordion} from './';
import renderer from 'react-test-renderer';
import {Client as Styletron} from 'styletron-engine-atomic';
import {Provider as StyletronProvider} from 'styletron-react';

const engine = new Styletron();

test('Accordion changes the class when hovered', () => {
  const component = renderer.create(
    <StyletronProvider value={engine}>
      <Accordion>Facebook</Accordion>,
    </StyletronProvider>
  );
  console.log(component);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});