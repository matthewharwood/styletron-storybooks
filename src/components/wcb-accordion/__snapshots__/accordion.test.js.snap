// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordion changes the class when hovered 1`] = `
Array [
  <div
    className="ae"
  >
    Foo
  </div>,
  ",",
]
`;
